src/scripts/upload-wpt-results.py

```diff
@@ -72,13 +68,16 @@ def main(raw_results_directory, product, browser_channel, browser_version,
                             'os_version': os_version
                         })
 
-            metadata['run_info'] = json.dumps(metadata['run_info'])
+            metadata_json = '"run_info":{}'.format(
```

Comment:
This value is not JSON, so I'd prefer to avoid that term. I can't think of a good alternative, so maybe instead we can just write to the handle directly.

Reply:
Yeah it was a JSON in an earlier iteration. `metadata_bits`? `metadata_fields`?